app/src/main/java/tech/ula/utils/AssetDownloader.kt

```diff
@@ -228,6 +229,10 @@ class DownloadManagerWrapper(private val downloadManager: DownloadManager) {
         }
         return DownloadFailureLocalizationData(R.string.download_failure_reason_not_found)
     }
+
+    fun cancelAllDownloads(downloadIds: Set<Long>) {
+        downloadManager.remove(*downloadIds.toLongArray())
```

Comment:
What does the asterisk signify here?

Reply:
It's a spread operator. `DownloadManager#remove` takes varargs.